src/gluonts/nursery/sagemaker_sdk/estimator.py

```diff
@@ -528,7 +528,7 @@ def train(
         wait: bool = True,
         logs: bool = True,
         job_name: str = None,
-    ) -> Tuple[Predictor, dict, pd.DataFrame, str]:
+    ) -> Optional[Tuple[Predictor, dict, pd.DataFrame, str]]:
```

Comment:
I think we always should return something. How do I know the training-job-name if I only the base_job_name?

Also, the return type looks off, shouldn't it be a `TrainResult`?

Reply:
Could be. A lot of things changed after the refactoring so I will have too look that up.

Reply:
Also you're right, we could always return the complete job name.